src/previewSupport/handleSourceUpdates.js

```diff
@@ -0,0 +1,8 @@
+import channel from './channel';
+
+export default function handleSourceUpdates() {
+  channel.bind('updateSrc', (_trans, source) => {
+    document.open();
+    document.write(source);
```

Comment:
I would throw a [`document.close()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/close) in here as well.